libraries/AP_HAL_ChibiOS/hwdef/ARKV6X/hwdef.dat

```diff
@@ -0,0 +1,335 @@
+# hw definition file for processing by chibios_hwdef.py
+# for the ARKV6X hardware
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 2
+
+# board ID for firmware load
+APJ_BOARD_ID 57
+
+# enable board sub-type detection for FMUV6 TODO: Figure out if this is needed for ARKV6X
```

Comment:
its not needed as you use a separate board ID. This is just used to distinguish the Holybro and CUAV 6X variants